config/lib/mongoose.js

```diff
@@ -24,25 +25,28 @@ module.exports.connect = function (cb) {
 
   mongoose.Promise = config.db.promise;
 
-  var db = mongoose.connect(config.db.uri, config.db.options, function (err) {
-    // Log Error
-    if (err) {
-      console.error(chalk.red('Could not connect to MongoDB!'));
-      console.log(err);
-    } else {
+  var options = _.merge(config.db.options || {}, { useMongoClient: true });
 
+  mongoose
+    .connect(config.db.uri, options)
+    .then(function (connection) {
       // Enabling mongoose debug mode if required
       mongoose.set('debug', config.db.debug);
 
       // Call callback FN
-      if (cb) cb(db);
-    }
-  });
+      if (cb) cb(connection.db);
```

Comment:
Personally I prefer using `callback` instead of `cb` just to keep it all the same and I dislike abbreviations in coding (just needs extra thinking when reading it), but this isn't a biggie. :-)